docs/analytics/api.mdx

```diff
@@ -8,17 +8,17 @@ For Sourcegraph Analytics, you can generate an access token for programmatic acc
 
 ### Getting Started
 
-Access tokens are created using the `cas` cookie for authentication to the token creation endpoint. Access tokens are longer lived than the `cas` cookie making them more suitable for programmatic access to the Sourcegraph Analytics API.
+Access tokens are created using the SAMS session cookie (`accounts_session_v2`) for authentication to the token creation endpoint. Access tokens are longer lived than the SAMS session cookie making them more suitable for programmatic access to the Sourcegraph Analytics API.
```

Comment:
Readers of this doc might not know what SAMS means, so let's use the full name at least once:

```suggestion
Access tokens are created using the Sourcegraph Accounts (SAMS) session cookie (`accounts_session_v2`) for authentication to the token creation endpoint. Access tokens are longer lived than the SAMS session cookie making them more suitable for programmatic access to the Sourcegraph Analytics API.
```